### ISSUE TITLE
This isn't working when you got + in your email for a login

### ISSUE THREAD

**Issue by codekhichdi** [riazXrazor/udemy-dl#70]
So I know a basics of node but kind of couldn't get how to build the code here..

so simple question, If I have modify the source and want to build and run it, what to do for that.?

**Comment by riazXrazor** [comment]
No building required, node bin/run :grin:

**Comment by codekhichdi** [comment]
Also one more thing, If I try to login it always says forbidden while I can login from the browser

**Comment by codekhichdi** [comment]
I got the + in my email , could that be an issue ?

**Comment by riazXrazor** [comment]
may be api has changed, or due to bot detection by them

**Comment by irfanjunaid** [comment]
Yep, it's not working for me either. Had been trying since last week.

**Error:**

```
$ udl login
https://www.udemy.com/join/login-popup/
Authencating... done
Forbidden
```

**Details:**
```
$ node -v
v12.16.2

OS: Ubuntu 18.04, Windows 10
```

**Comment by AbreguWalter** [comment]
Mismo problema
$ udl login
https://www.udemy.com/join/login-popup/
Authencating... done
Forbidden

**Comment by irfanjunaid** [comment]
Hi @riazXrazor  Any update? We still cannot use the script...

**Comment by riazXrazor** [comment]
the script is being clocked by cloudflare detcting it as bot, 
sorry guys no solution yet.
